ispyb tests failing on main

Some ispyb tests are failing on main:
```
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids0] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids1] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids2] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids3] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids4] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids5] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids6] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_reuses_dcgid_on_same_sampleID[ispyb_ids7] - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py::test_ispyb_specifies_experiment_type_if_supplied - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds
FAILED tests/unit_tests/external_interaction/ispyb/test_rotation_ispyb_store.py::test_begin_deposition_with_alternate_experiment_type - pydantic.error_wrappers.ValidationError: 1 validation error for IspybIds

```
## Acceptance Criteria
* Fix them

--- a/tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py
+++ b/tests/unit_tests/external_interaction/callbacks/test_rotation_callbacks.py
@@ -334,7 +334,7 @@ def test_ispyb_reuses_dcgid_on_same_sampleID(
     cb = [RotationISPyBCallback()]
     cb[0].active = True
     ispyb_ids = IspybIds(
-        data_collection_group_id=23, data_collection_ids=45, grid_ids=None
+        data_collection_group_id=23, data_collection_ids=(45,), grid_ids=None
     )
     rotation_ispyb.return_value.begin_deposition.return_value = ispyb_ids
 
@@ -378,7 +378,7 @@ def test_ispyb_specifies_experiment_type_if_supplied(
     cb[0].active = True
     params.hyperion_params.ispyb_params.ispyb_experiment_type = "Characterization"
     rotation_ispyb.return_value.begin_deposition.return_value = IspybIds(
-        data_collection_group_id=23, data_collection_ids=45, grid_ids=None
+        data_collection_group_id=23, data_collection_ids=(45,), grid_ids=None
     )
 
     params.hyperion_params.ispyb_params.sample_id = "abc"

--- a/tests/unit_tests/external_interaction/ispyb/test_rotation_ispyb_store.py
+++ b/tests/unit_tests/external_interaction/ispyb/test_rotation_ispyb_store.py
@@ -118,7 +118,7 @@ def test_begin_deposition_with_alternate_experiment_type(
     dummy_rotation_params,
 ):
     assert dummy_rotation_ispyb_with_experiment_type.begin_deposition() == IspybIds(
-        data_collection_ids=TEST_DATA_COLLECTION_IDS[0],
+        data_collection_ids=(TEST_DATA_COLLECTION_IDS[0],),
         data_collection_group_id=TEST_DATA_COLLECTION_GROUP_ID,
     )
     mx_acq = mx_acquisition_from_conn(ispyb_conn_with_2x2_collections_and_grid_info)

